test/test_playbooks/upload.yml

```diff
@@ -34,10 +34,6 @@
   - name: 'Load server config'
     include_vars:
       file: server_vars.yml
-  - name: remove RPM
```

Comment:
@sean797 I removed this, as the RPM is in the repo, and removing it makes the test not re-runable w/o restoring the file.

Reply:
Does this mean, we are dropping a test here? Can we readd it / Sync the repo afterwards?

Reply:
No we are not!

`upload.yml` is about uploading a file (in this case an RPM) to Katello. The RPM used for this is checked in into Git as a fixture: https://github.com/theforeman/foreman-ansible-modules/blob/master/test/test_playbooks/data/bear-4.1-1.noarch.rpm

I've just modified the playbook not to remove the fixture file, as this seems contra-productive here as it requires you to fiddle with Git before you can run `upload.yml` again.

It would be correct to remove it if we'd fetch it dynamically from a remote site before uploading, but we're not doing that.